### DISCUSSION TITLE
Documentation link doesn't work

### DISCUSSION THREAD

**Discussion by yevhen-ikim** [Finbuckle/Finbuckle.MultiTenant#858] [General]
Version v7.0.1.

The Documentation link on GitHub's Documentation section doesn't work. 
![image](https://github.com/user-attachments/assets/31a7ddc8-506f-4627-b8d4-26cd612d3c73)

The same for the Documentation link at the bottom of the website page. 
![image](https://github.com/user-attachments/assets/831a958f-3a5c-4a82-8bd3-a0c081be3d95)

Error:
![image](https://github.com/user-attachments/assets/39957a92-875c-4737-887d-f19677ddc9f5)

**Comment by AndrewTriesToCode** [comment]
Thanks for letting me know!

**Comment by AndrewTriesToCode** [comment]
Should be fixed now. Thanks again.
